.ddev/ci/scripts/kafka_consumer/linux/32_install_kerberos.sh

```diff
@@ -4,6 +4,12 @@ set -ex
 
 sudo apt-get update
 sudo apt install build-essential
-sudo apt-get install -y --no-install-recommends libkrb5-dev
+sudo apt-get install -y --no-install-recommends libkrb5-dev wget software-properties-common lsb-release gcc make python3 python3-pip python3-dev libsasl2-modules-gssapi-mit krb5-user
+
+# Install the latest version of librdkafka:
+wget -qO - https://packages.confluent.io/deb/7.3/archive.key | sudo apt-key add -
+sudo add-apt-repository "deb https://packages.confluent.io/clients/deb $(lsb_release -cs) main"
+sudo apt update
+sudo apt install -y librdkafka-dev
```

Comment:
It does not mean that these changes become unnecessary though. This script install the deps in the runner to be able to use kerberos in integration tests (so running the code from the runner).

Reply:
ah but https://github.com/DataDog/integrations-core/pull/13918/files#r1164528429, so yes I guess we can get rid of this in this case

Reply:
Sorry but in fact we can't because we have some code in the conditions that are interacting with kafka itself, which means we need to support kerberos on the CI itself. I opened a draft PR to fix that (because it's currently failing on master)

Reply:
ftr: https://github.com/DataDog/integrations-core/pull/14390